HyperwalletUISDK.xcodeproj/xcshareddata/xcschemes/HyperwalletUISDK.xcscheme

```diff
@@ -26,6 +26,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      region = "US"
```

Comment:
Why added this?

Reply:
our tests depend on the locale. Some tests on my Ukrainian locale are failed

Reply:
reverted